javacutil/src/main/java/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -447,6 +447,9 @@ public static boolean hasInheritedMeta(AnnotationMirror anno) {
             case TYPE_USE:
                 return EnumSet.noneOf(ElementKind.class);
             default:
+                if (elementType.name().contentEquals("MODULE")) {
```

Comment:
It would be good to document this in the method javadoc.
Is this a temporary workaround or should this method never support `MODULE`?

Reply:
This temporary, which I've now noted above.

Reply:
This is also caused by source level 8. It is a bit surprising that the source level has an impact on this, but when I change the build to not pass `source 8` I can directly refer to `ElementKind.MODULE` in the switch.
As we still want to stay on Java 8, I think this is fine, but should be documented here.

Reply:
It's already appropriately documented.